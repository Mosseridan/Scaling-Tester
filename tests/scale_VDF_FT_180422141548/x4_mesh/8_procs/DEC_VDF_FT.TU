Statistiques de post resolution

Temps total                       0.473079

