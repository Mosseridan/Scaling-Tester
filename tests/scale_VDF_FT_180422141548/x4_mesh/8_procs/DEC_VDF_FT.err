Petsc initialization succeeded.
Parallel engine initialized : Comm_Group_Noparallel with 1 processors
Executable: /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
Data file name = DEC_VDF_FT
feenableexcept enabled.
Data file : DEC_VDF_FT
Localisation etude: /home/idanmos/projects/Scaling-Tester/tests/scale_VDF_FT_180422-141548/x4_mesh/8_procs
Nom du cas DEC_VDF_FT
 code : /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
 version : 1.7.6 
Debut de l'execution 
MAIN: Reading and executing data file
Dimension::interpreter : dimension = 3
Execution of the interpreter Mailler
Association of the domain on which the interpreter applies Mailler
Reading of the block pave1
Step of 3D mesh in progress... 
It was created 5% of coordxIt was created 10% of coordxIt was created 15% of coordxIt was created 20% of coordxIt was created 25% of coordxIt was created 30% of coordxIt was created 35% of coordxIt was created 40% of coordxIt was created 45% of coordxIt was created 50% of coordxIt was created 55% of coordxIt was created 60% of coordxIt was created 65% of coordxIt was created 70% of coordxIt was created 75% of coordxIt was created 80% of coordxIt was created 85% of coordxIt was created 90% of coordxIt was created 95% of coordx
It was created 5% of coordyIt was created 10% of coordyIt was created 15% of coordyIt was created 20% of coordyIt was created 25% of coordyIt was created 30% of coordyIt was created 35% of coordyIt was created 40% of coordyIt was created 45% of coordyIt was created 50% of coordyIt was created 55% of coordyIt was created 60% of coordyIt was created 65% of coordyIt was created 70% of coordyIt was created 75% of coordyIt was created 80% of coordyIt was created 85% of coordyIt was created 90% of coordyIt was created 95% of coordy
It was created 2% of coordzIt was created 5% of coordzIt was created 7% of coordzIt was created 10% of coordzIt was created 12% of coordzIt was created 15% of coordzIt was created 17% of coordzIt was created 20% of coordzIt was created 22% of coordzIt was created 25% of coordzIt was created 27% of coordzIt was created 30% of coordzIt was created 32% of coordzIt was created 35% of coordzIt was created 37% of coordzIt was created 40% of coordzIt was created 42% of coordzIt was created 45% of coordzIt was created 47% of coordzIt was created 50% of coordzIt was created 52% of coordzIt was created 55% of coordzIt was created 57% of coordzIt was created 60% of coordzIt was created 62% of coordzIt was created 65% of coordzIt was created 67% of coordzIt was created 70% of coordzIt was created 72% of coordzIt was created 75% of coordzIt was created 77% of coordzIt was created 80% of coordzIt was created 82% of coordzIt was created 85% of coordzIt was created 87% of coordzIt was created 90% of coordzIt was created 92% of coordzIt was created 95% of coordzIt was created 97% of coordz
Step of mesh ended 
Reading of the boundary paroi1
End of reading of the boundary : paroi1
Reading of the boundary haut1
End of reading of the boundary : haut1
Reading of the boundary bas1
End of reading of the boundary : bas1
Reading of the boundary paroi1
End of reading of the boundary : paroi1
Reading of the boundary paroi1
End of reading of the boundary : paroi1
Reading of the boundary paroi1
End of reading of the boundary : paroi1
Compressing objects of type Zone in progress... 
Zones::comprimer() - End
Elimination of 0 unnecessary nodes in the domain dom
Execution of the interpreter Transformer
Association of the domain on which the interpreter applies Transformer
Reading and interpretation of the function x*0.008-0.004
Interpretation of the function x*0.008-0.004 Ok
Reading and interpretation of the function y*0.008-0.004
Interpretation of the function y*0.008-0.004 Ok
Reading and interpretation of the function z*0.016
Interpretation of the function z*0.016 Ok
Hexaedre::reordonner : 0 elements reversed
Hexaedre::reordonner : 0 elements reversed
Execution of the interpreter Mailler
Association of the domain on which the interpreter applies Mailler
Reading of the block pave2
Step of 3D mesh in progress... 
It was created 5% of coordxIt was created 10% of coordxIt was created 15% of coordxIt was created 20% of coordxIt was created 25% of coordxIt was created 30% of coordxIt was created 35% of coordxIt was created 40% of coordxIt was created 45% of coordxIt was created 50% of coordxIt was created 55% of coordxIt was created 60% of coordxIt was created 65% of coordxIt was created 70% of coordxIt was created 75% of coordxIt was created 80% of coordxIt was created 85% of coordxIt was created 90% of coordxIt was created 95% of coordx
It was created 5% of coordyIt was created 10% of coordyIt was created 15% of coordyIt was created 20% of coordyIt was created 25% of coordyIt was created 30% of coordyIt was created 35% of coordyIt was created 40% of coordyIt was created 45% of coordyIt was created 50% of coordyIt was created 55% of coordyIt was created 60% of coordyIt was created 65% of coordyIt was created 70% of coordyIt was created 75% of coordyIt was created 80% of coordyIt was created 85% of coordyIt was created 90% of coordyIt was created 95% of coordy
It was created 50% of coordz
Step of mesh ended 
Reading of the boundary paroi2
End of reading of the boundary : paroi2
Reading of the boundary haut2
End of reading of the boundary : haut2
Reading of the boundary bas2
End of reading of the boundary : bas2
Reading of the boundary paroi2
End of reading of the boundary : paroi2
Reading of the boundary paroi2
End of reading of the boundary : paroi2
Reading of the boundary paroi2
End of reading of the boundary : paroi2
Compressing objects of type Zone in progress... 
Zones::comprimer() - End
Elimination of 0 unnecessary nodes in the domain dom_solide
Execution of the interpreter Transformer
Association of the domain on which the interpreter applies Transformer
Reading and interpretation of the function x*0.008-0.004
Interpretation of the function x*0.008-0.004 Ok
Reading and interpretation of the function y*0.008-0.004
Interpretation of the function y*0.008-0.004 Ok
Reading and interpretation of the function z*0.0001-0.0001
Interpretation of the function z*0.0001-0.0001 Ok
Hexaedre::reordonner : 0 elements reversed
Hexaedre::reordonner : 0 elements reversed
Decouper : Splitting of a domain
 Domain name to split : dom
==============================================
The extreme coordinates of the domain dom are:
x is between -0.004 and 0.004
y is between -0.004 and 0.004
z is between 0 and 0.016
==============================================
==============================================
The volume cells of the domain dom are:
sum(volume cells)= 1.024e-06
mean(volume cells)= 6.4e-11
min(volume cells)= 6.4e-11
max(volume cells)= 6.4e-11
==============================================
domaine = dom
 Creation of a partitioner of type: Partitionneur_metis
Reading parameters of Partitionneur_Metis
Partitionneur_Metis::construire_partition
 Construction of graph connectivity...
 Construction of the som_elem connectivity
 Construction of the elem_elem connectivity
===============
Call for PMETIS
===============
===============
Partitioning quality : edgecut = 2040
-> It is roughly the total number of edges (faces) which will be shared by the processors.
-> The lesser this number is, the lesser the total volume of communication between processors.
-> You can increase nb_essais option (default 1) to try to reduce (but at a higher CPU cost) this number.
===============
Correction elem0 on processor 0
Correction of the splitting to put the element 0 on processor 0.
 Exchange of parts 0 and 7
The partitioner has generated 8 parts.
Number of parts requested : 8
Generation of 0 empty parts.
Search of neighboring elements of boundary faces
Construction of lists of elements per subdomain
Generation of 8 parts:
 Construction of part number 0
  Number of nodes    : 2541
  Number of elements : 2000
  Number of joints   : 3
  Joint 0 PeVoisin 4 NbSommets 215 NbFaces 184 NbElemDist 400
  Joint 1 PeVoisin 5 NbSommets 29 NbFaces 16 NbElemDist 96
  Joint 2 PeVoisin 6 NbSommets 231 NbFaces 200 NbElemDist 400
               Total:    NbSommets 475 NbFaces 400 NbElemDist 896
Writing part 0 into the binary file DOM_0000.Zones
Writing of 2541 nodes.
Writing of 2000 elements.
 Construction of part number 1
  Number of nodes    : 2541
  Number of elements : 2000
  Number of joints   : 3
  Joint 0 PeVoisin 2 NbSommets 21 NbFaces 0 NbElemDist 80
  Joint 1 PeVoisin 3 NbSommets 231 NbFaces 200 NbElemDist 400
  Joint 2 PeVoisin 7 NbSommets 231 NbFaces 200 NbElemDist 400
               Total:    NbSommets 483 NbFaces 400 NbElemDist 880
Writing part 1 into the binary file DOM_0001.Zones
Writing of 2541 nodes.
Writing of 2000 elements.
 Construction of part number 2
  Number of nodes    : 2541
  Number of elements : 2000
  Number of joints   : 5
  Joint 0 PeVoisin 1 NbSommets 21 NbFaces 0 NbElemDist 80
  Joint 1 PeVoisin 3 NbSommets 231 NbFaces 200 NbElemDist 400
  Joint 2 PeVoisin 4 NbSommets 30 NbFaces 16 NbElemDist 96
  Joint 3 PeVoisin 5 NbSommets 215 NbFaces 184 NbElemDist 400
  Joint 4 PeVoisin 7 NbSommets 231 NbFaces 200 NbElemDist 400
               Total:    NbSommets 728 NbFaces 600 NbElemDist 1376
Writing part 2 into the binary file DOM_0002.Zones
Writing of 2541 nodes.
Writing of 2000 elements.
 Construction of part number 3
  Number of nodes    : 2541
  Number of elements : 2000
  Number of joints   : 5
  Joint 0 PeVoisin 1 NbSommets 231 NbFaces 200 NbElemDist 400
  Joint 1 PeVoisin 2 NbSommets 231 NbFaces 200 NbElemDist 400
  Joint 2 PeVoisin 4 NbSommets 215 NbFaces 184 NbElemDist 400
  Joint 3 PeVoisin 5 NbSommets 29 NbFaces 16 NbElemDist 96
  Joint 4 PeVoisin 7 NbSommets 21 NbFaces 0 NbElemDist 80
               Total:    NbSommets 727 NbFaces 600 NbElemDist 1376
Writing part 3 into the binary file DOM_0003.Zones
Writing of 2541 nodes.
Writing of 2000 elements.
 Construction of part number 4
  Number of nodes    : 2563
  Number of elements : 2000
  Number of joints   : 5
  Joint 0 PeVoisin 0 NbSommets 215 NbFaces 184 NbElemDist 400
  Joint 1 PeVoisin 2 NbSommets 30 NbFaces 16 NbElemDist 80
  Joint 2 PeVoisin 3 NbSommets 215 NbFaces 184 NbElemDist 400
  Joint 3 PeVoisin 5 NbSommets 275 NbFaces 240 NbElemDist 480
  Joint 4 PeVoisin 6 NbSommets 30 NbFaces 16 NbElemDist 80
               Total:    NbSommets 765 NbFaces 640 NbElemDist 1440
Writing part 4 into the binary file DOM_0004.Zones
Writing of 2563 nodes.
Writing of 2000 elements.
 Construction of part number 5
  Number of nodes    : 2563
  Number of elements : 2000
  Number of joints   : 5
  Joint 0 PeVoisin 0 NbSommets 29 NbFaces 16 NbElemDist 80
  Joint 1 PeVoisin 2 NbSommets 215 NbFaces 184 NbElemDist 400
  Joint 2 PeVoisin 3 NbSommets 29 NbFaces 16 NbElemDist 80
  Joint 3 PeVoisin 4 NbSommets 275 NbFaces 240 NbElemDist 480
  Joint 4 PeVoisin 6 NbSommets 215 NbFaces 184 NbElemDist 400
               Total:    NbSommets 763 NbFaces 640 NbElemDist 1440
Writing part 5 into the binary file DOM_0005.Zones
Writing of 2563 nodes.
Writing of 2000 elements.
 Construction of part number 6
  Number of nodes    : 2541
  Number of elements : 2000
  Number of joints   : 3
  Joint 0 PeVoisin 0 NbSommets 231 NbFaces 200 NbElemDist 400
  Joint 1 PeVoisin 4 NbSommets 30 NbFaces 16 NbElemDist 96
  Joint 2 PeVoisin 5 NbSommets 215 NbFaces 184 NbElemDist 400
               Total:    NbSommets 476 NbFaces 400 NbElemDist 896
Writing part 6 into the binary file DOM_0006.Zones
Writing of 2541 nodes.
Writing of 2000 elements.
 Construction of part number 7
  Number of nodes    : 2541
  Number of elements : 2000
  Number of joints   : 3
  Joint 0 PeVoisin 1 NbSommets 231 NbFaces 200 NbElemDist 400
  Joint 1 PeVoisin 2 NbSommets 231 NbFaces 200 NbElemDist 400
  Joint 2 PeVoisin 3 NbSommets 21 NbFaces 0 NbElemDist 80
               Total:    NbSommets 483 NbFaces 400 NbElemDist 880
Writing part 7 into the binary file DOM_0007.Zones
Writing of 2541 nodes.
Writing of 2000 elements.
End of the interpreter Decouper
Decouper : Splitting of a domain
 Domain name to split : dom_solide
==============================================
The extreme coordinates of the domain dom_solide are:
x is between -0.004 and 0.004
y is between -0.004 and 0.004
z is between -0.0001 and 0
==============================================
==============================================
The volume cells of the domain dom_solide are:
sum(volume cells)= 6.4e-09
mean(volume cells)= 8e-12
min(volume cells)= 8e-12
max(volume cells)= 8e-12
==============================================
domaine = dom_solide
 Creation of a partitioner of type: Partitionneur_metis
Reading parameters of Partitionneur_Metis
Partitionneur_Metis::construire_partition
 Construction of graph connectivity...
 Construction of the som_elem connectivity
 Construction of the elem_elem connectivity
===============
Call for PMETIS
===============
===============
Partitioning quality : edgecut = 178
-> It is roughly the total number of edges (faces) which will be shared by the processors.
-> The lesser this number is, the lesser the total volume of communication between processors.
-> You can increase nb_essais option (default 1) to try to reduce (but at a higher CPU cost) this number.
===============
Correction elem0 on processor 0
Correction of the splitting to put the element 0 on processor 0.
 Exchange of parts 0 and 7
The partitioner has generated 8 parts.
Search of neighboring elements of boundary faces
Construction of lists of elements per subdomain
Generation of 8 parts:
 Construction of part number 0
  Number of nodes    : 201
  Number of elements : 100
  Number of joints   : 2
  Joint 0 PeVoisin 3 NbSommets 21 NbFaces 12 NbElemDist 24
  Joint 1 PeVoisin 6 NbSommets 33 NbFaces 20 NbElemDist 40
               Total:    NbSommets 54 NbFaces 32 NbElemDist 64
Writing part 0 into the binary file DOM_SOLIDE_0000.Zones
Writing of 201 nodes.
Writing of 100 elements.
 Construction of part number 1
 Adding of a new joint : 6
  Number of nodes    : 204
  Number of elements : 100
  Number of joints   : 5
  Joint 0 PeVoisin 2 NbSommets 18 NbFaces 10 NbElemDist 28
  Joint 1 PeVoisin 3 NbSommets 24 NbFaces 14 NbElemDist 34
  Joint 2 PeVoisin 5 NbSommets 18 NbFaces 10 NbElemDist 24
  Joint 3 PeVoisin 6 NbSommets 0 NbFaces 0 NbElemDist 2
  Joint 4 PeVoisin 7 NbSommets 42 NbFaces 26 NbElemDist 48
               Total:    NbSommets 102 NbFaces 60 NbElemDist 136
Writing part 1 into the binary file DOM_SOLIDE_0001.Zones
Writing of 204 nodes.
Writing of 100 elements.
 Construction of part number 2
  Number of nodes    : 201
  Number of elements : 100
  Number of joints   : 2
  Joint 0 PeVoisin 1 NbSommets 18 NbFaces 10 NbElemDist 20
  Joint 1 PeVoisin 3 NbSommets 36 NbFaces 22 NbElemDist 44
               Total:    NbSommets 54 NbFaces 32 NbElemDist 64
Writing part 2 into the binary file DOM_SOLIDE_0002.Zones
Writing of 201 nodes.
Writing of 100 elements.
 Construction of part number 3
  Number of nodes    : 204
  Number of elements : 100
  Number of joints   : 5
  Joint 0 PeVoisin 0 NbSommets 21 NbFaces 12 NbElemDist 32
  Joint 1 PeVoisin 1 NbSommets 24 NbFaces 14 NbElemDist 32
  Joint 2 PeVoisin 2 NbSommets 36 NbFaces 22 NbElemDist 44
  Joint 3 PeVoisin 5 NbSommets 9 NbFaces 4 NbElemDist 12
  Joint 4 PeVoisin 6 NbSommets 18 NbFaces 10 NbElemDist 32
               Total:    NbSommets 108 NbFaces 62 NbElemDist 152
Writing part 3 into the binary file DOM_SOLIDE_0003.Zones
Writing of 204 nodes.
Writing of 100 elements.
 Construction of part number 4
  Number of nodes    : 198
  Number of elements : 100
  Number of joints   : 2
  Joint 0 PeVoisin 5 NbSommets 33 NbFaces 20 NbElemDist 40
  Joint 1 PeVoisin 6 NbSommets 18 NbFaces 10 NbElemDist 20
               Total:    NbSommets 51 NbFaces 30 NbElemDist 60
Writing part 4 into the binary file DOM_SOLIDE_0004.Zones
Writing of 198 nodes.
Writing of 100 elements.
 Construction of part number 5
  Number of nodes    : 201
  Number of elements : 100
  Number of joints   : 5
  Joint 0 PeVoisin 1 NbSommets 18 NbFaces 10 NbElemDist 30
  Joint 1 PeVoisin 3 NbSommets 9 NbFaces 4 NbElemDist 18
  Joint 2 PeVoisin 4 NbSommets 33 NbFaces 20 NbElemDist 40
  Joint 3 PeVoisin 6 NbSommets 18 NbFaces 10 NbElemDist 22
  Joint 4 PeVoisin 7 NbSommets 18 NbFaces 10 NbElemDist 28
               Total:    NbSommets 96 NbFaces 54 NbElemDist 138
Writing part 5 into the binary file DOM_SOLIDE_0005.Zones
Writing of 201 nodes.
Writing of 100 elements.
 Construction of part number 6
 Adding of a new joint : 1
  Number of nodes    : 198
  Number of elements : 100
  Number of joints   : 5
  Joint 0 PeVoisin 0 NbSommets 33 NbFaces 20 NbElemDist 40
  Joint 1 PeVoisin 1 NbSommets 0 NbFaces 0 NbElemDist 2
  Joint 2 PeVoisin 3 NbSommets 18 NbFaces 10 NbElemDist 20
  Joint 3 PeVoisin 4 NbSommets 18 NbFaces 10 NbElemDist 28
  Joint 4 PeVoisin 5 NbSommets 18 NbFaces 10 NbElemDist 28
               Total:    NbSommets 87 NbFaces 50 NbElemDist 118
Writing part 6 into the binary file DOM_SOLIDE_0006.Zones
Writing of 198 nodes.
Writing of 100 elements.
 Construction of part number 7
  Number of nodes    : 204
  Number of elements : 100
  Number of joints   : 2
  Joint 0 PeVoisin 1 NbSommets 42 NbFaces 26 NbElemDist 52
  Joint 1 PeVoisin 5 NbSommets 18 NbFaces 10 NbElemDist 20
               Total:    NbSommets 60 NbFaces 36 NbElemDist 72
Writing part 7 into the binary file DOM_SOLIDE_0007.Zones
Writing of 204 nodes.
Writing of 100 elements.
End of the interpreter Decouper
MAIN: End of data file
Arret des processes.
