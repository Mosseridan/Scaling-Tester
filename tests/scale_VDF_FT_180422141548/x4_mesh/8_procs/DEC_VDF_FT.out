-------------------------------------------------------------------
 
                          TRUST
                      version : 1.7.6 
                          CEA - DEN
 

 0 file(s) modified 


 405 file(s) created 

 ------------------------------------------------------------------
 
  Vous traitez le cas DEC_VDF_FT
 
Reading data file 

14 MBytes of RAM taken by the calculation.
15 MBytes of RAM taken by the calculation.
16 MBytes of RAM taken by the calculation.
16 MBytes of RAM taken by the calculation.
 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 8
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[1.25e-01]   1=[1.25e-01]   2=[1.25e-01]   3=[1.25e-01]   4=[1.25e-01]
        5=[1.25e-01]   6=[1.25e-01]   7=[1.25e-01]
   Allowed maximum load imbalance: 1.001 

     16000      92000      92000 [20] [     1200:   16000 ]
      8371      67350      76742 [20] [     1200:   16000 ]
      4383      43828      64964 [20] [     1200:   16000 ]
      2296      26120      55448 [20] [     1200:   16000 ]
      1194      14656      46956 [20] [     1200:   16000 ]
       622       7840      39796 [20] [     1200:   16000 ]
       322       4070      33642 [20] [     1200:   16000 ]
       165       2004      28164 [20] [     1200:   16000 ]
        84        964      23324 [20] [     1200:   16000 ]
        44        460      19158 [20] [     1200:   16000 ]
        23        194      15168 [20] [     1200:   16000 ]
        20        158      14698 [20] [     1200:   16000 ]
      8000      45600      45600 [20] [      600:    8000 ]
      4184      33326      37968 [20] [      600:    8000 ]
      2197      21752      32068 [20] [      600:    8000 ]
      1150      12934      27282 [20] [      600:    8000 ]
       600       7210      23056 [20] [      600:    8000 ]
       311       3824      19526 [20] [      600:    8000 ]
       162       1958      16338 [20] [      600:    8000 ]
        85        946      13664 [20] [      600:    8000 ]
        45        444      11294 [20] [      600:    8000 ]
        23        196       9196 [20] [      600:    8000 ]
        18        146       8818 [20] [      600:    8000 ]
      4000      22400      22400 [20] [      300:    4000 ]
      2093      16284      18586 [20] [      300:    4000 ]
      1107      10502      15690 [20] [      300:    4000 ]
       579       6176      13280 [20] [      300:    4000 ]
       304       3402      11216 [20] [      300:    4000 ]
       160       1820       9480 [20] [      300:    4000 ]
        82        894       7844 [20] [      300:    4000 ]
        44        428       6404 [20] [      300:    4000 ]
        23        194       5200 [20] [      300:    4000 ]
        19        154       4984 [20] [      300:    4000 ]
      4000      22400      22400 [20] [      300:    4000 ]
      2075      16156      18550 [20] [      300:    4000 ]
      1090      10468      15622 [20] [      300:    4000 ]
       569       6152      13204 [20] [      300:    4000 ]
       298       3344      11082 [20] [      300:    4000 ]
       158       1744       9250 [20] [      300:    4000 ]
        83        866       7756 [20] [      300:    4000 ]
        42        400       6304 [20] [      300:    4000 ]
        22        174       5006 [20] [      300:    4000 ]
        20        158       4924 [20] [      300:    4000 ]
      8000      45600      45600 [20] [      600:    8000 ]
      4178      33488      37956 [20] [      600:    8000 ]
      2193      21784      32140 [20] [      600:    8000 ]
      1156      13020      27366 [20] [      600:    8000 ]
       606       7196      23128 [20] [      600:    8000 ]
       315       3854      19570 [20] [      600:    8000 ]
       164       1956      16492 [20] [      600:    8000 ]
        88        966      13728 [20] [      600:    8000 ]
        47        466      11432 [20] [      600:    8000 ]
        25        216       9502 [20] [      600:    8000 ]
        18        146       8530 [20] [      600:    8000 ]
      4000      22400      22400 [20] [      300:    4000 ]
      2097      16334      18594 [20] [      300:    4000 ]
      1102      10550      15654 [20] [      300:    4000 ]
       579       6206      13268 [20] [      300:    4000 ]
       301       3432      11202 [20] [      300:    4000 ]
       160       1846       9434 [20] [      300:    4000 ]
        84        922       7848 [20] [      300:    4000 ]
        43        444       6554 [20] [      300:    4000 ]
        23        212       5480 [20] [      300:    4000 ]
        19        166       5326 [20] [      300:    4000 ]
      4000      22400      22400 [20] [      300:    4000 ]
      2093      16270      18586 [20] [      300:    4000 ]
      1102      10662      15666 [20] [      300:    4000 ]
       577       6278      13340 [20] [      300:    4000 ]
       300       3406      11242 [20] [      300:    4000 ]
       158       1814       9470 [20] [      300:    4000 ]
        82        914       7876 [20] [      300:    4000 ]
        42        432       6430 [20] [      300:    4000 ]
        22        198       5252 [20] [      300:    4000 ]
        19        166       5070 [20] [      300:    4000 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.051
     Coarsening: 		   0.027
            Matching: 			   0.006
            Contract: 			   0.020
     Initial Partition: 	   0.001
     Uncoarsening: 		   0.023
          Refinement: 			   0.017
          Projection: 			   0.006
     Splitting: 		   0.000
********************************************************************

 gk_mcore statistics
           coresize:       192552         nmops:         2048  cmop:      0
        num_callocs:          741   num_hallocs:            0
       size_callocs:      2736816  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:       192016   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 8
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[1.25e-01]   1=[1.25e-01]   2=[1.25e-01]   3=[1.25e-01]   4=[1.25e-01]
        5=[1.25e-01]   6=[1.25e-01]   7=[1.25e-01]
   Allowed maximum load imbalance: 1.001 

       800       3840       3840 [20] [       60:     800 ]
       420       2660       3080 [20] [       60:     800 ]
       220       1594       2488 [20] [       60:     800 ]
       116        830       2020 [20] [       60:     800 ]
        63        396       1612 [20] [       60:     800 ]
        33        174       1166 [20] [       60:     800 ]
        19         84        888 [20] [       60:     800 ]
       400       1876       1876 [20] [       30:     400 ]
       212       1294       1500 [20] [       30:     400 ]
       113        784       1212 [20] [       30:     400 ]
        61        410        968 [20] [       30:     400 ]
        32        190        728 [20] [       30:     400 ]
        18         84        554 [20] [       30:     400 ]
       200        916        916 [20] [       15:     200 ]
       105        622        726 [20] [       15:     200 ]
        55        344        580 [20] [       15:     200 ]
        29        172        448 [20] [       15:     200 ]
        19        104        382 [20] [       15:     200 ]
       200        912        912 [20] [       15:     200 ]
       104        610        720 [20] [       15:     200 ]
        55        344        574 [20] [       15:     200 ]
        29        168        444 [20] [       15:     200 ]
        16         82        344 [20] [       15:     200 ]
       400       1872       1872 [20] [       30:     400 ]
       211       1294       1494 [20] [       30:     400 ]
       115        816       1220 [20] [       30:     400 ]
        60        414        976 [20] [       30:     400 ]
        33        188        768 [20] [       30:     400 ]
        17         76        532 [20] [       30:     400 ]
       200        916        916 [20] [       15:     200 ]
       106        626        728 [20] [       15:     200 ]
        56        370        584 [20] [       15:     200 ]
        30        182        458 [20] [       15:     200 ]
        17         94        350 [20] [       15:     200 ]
       200        916        916 [20] [       15:     200 ]
       104        612        724 [20] [       15:     200 ]
        56        372        582 [20] [       15:     200 ]
        29        168        468 [20] [       15:     200 ]
        18         90        382 [20] [       15:     200 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.005
     Coarsening: 		   0.002
            Matching: 			   0.001
            Contract: 			   0.001
     Initial Partition: 	   0.001
     Uncoarsening: 		   0.002
          Refinement: 			   0.001
          Projection: 			   0.001
     Splitting: 		   0.000
********************************************************************

 gk_mcore statistics
           coresize:        10152         nmops:         2048  cmop:      0
        num_callocs:          430   num_hallocs:            0
       size_callocs:       148168  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:         9616   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

18 MBytes of RAM taken by the calculation.

--------------------------------------------
clock: Total execution: 0.473079 s
